app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,69 @@
+class Admin::UsersController < ApplicationController
+    before_action :set_user, only: [:show, :edit, :update, :destroy]
+    #before_action :require_login
+
+    def index
+      @users = User.all
+      unless current_user.admin?
+        redirect_to user_path(current_user), notice: '管理者権限ページへアクセスは出来ません!'
+      end
+    end
+
+    def create
+        @user = User.new(user_params)
+        if @user.save
+          redirect_to user_path(@user.id)
+        else
+          render :new
+        end
+    end
+
+    def new
+        @user = User.new
+    end
+
+    def edit
+    end
+
+    def show
+      @user = User.find(params[:id])
+      @tasks = @user.tasks
+    
+    end
+
+    def update
+      @user.update(user_params)
+      if User.where(admin: :true).count == 0
+        @user.update(admin: :true)
+        redirect_to admin_users_path
+        flash[:warning] = "管理者は編集できました"
+      elsif @user.save == false
+        flash[:danger] = "管理者は１名以下なので編集できませんでした"
+        render :edit
+      elsif User.where(admin: :true).count >= 1
```

Comment:
1件以上存在することの確認は下記の様にも記載できます。
```suggestion
      elsif User.where(admin: :true).any?
```